lib/internal/util.js

```diff
@@ -366,6 +366,14 @@ function isInsideNodeModules() {
   return false;
 }
 
+const kPropertyFilter = Object.freeze({
```

Comment:
Would it make sense to copy the whole enum as it is in V8 ( https://github.com/v8/v8/blob/master/src/property-details.h#L36-L45 ) for consistency and keeping a reference to the V8 source here?

Reply:
So far, we usually go all the way here and provide the real V8 constants as exports from the binding, e.g. https://github.com/nodejs/node/blob/9bcb744ee58f7f53f0a9b77f6ee36a4477ab43b4/src/pipe_wrap.cc#L105-L114

I’d encourage you to keep up with that pattern to prevent unexpected breakage from changes to the V8 definition, which is the source-of-truth for these values, even if it seems like a bit of extra work at this point (or leave to TODO comment for me, I’m happy to do the work myself if you prefer)